Bank/Helper.cs

```diff
@@ -0,0 +1,207 @@
+                    case BankOperations.UserOperations.RegisterAccount:
+                        RegisterAccount(accounts);
+                        break;
+                    case BankOperations.UserOperations.SimulateTimeShift:
+                        SimulateTimeShift(accounts);
+                        break;
+                    case BankOperations.UserOperations.Deposit:
+                        Deposit(accounts);
+                        break;
+                    case BankOperations.UserOperations.Withdraw:
+                        Withdraw(accounts);
+                        break;
+                    case BankOperations.UserOperations.PrintAccount:
+                        PrintAccount(accounts);
+                        break;
+                    case BankOperations.UserOperations.CloseAccount:
+                        CloseAccount(accounts);
+                        return;
+                    case BankOperations.UserOperations.Exit:
+                        return;
+                }
+            }
+        }
+        public static BankOperations.UserOperations GetUserValidChoice()
+        {
+            BankOperations.UserOperations operation;
+            PrintMenu();
+            while (true)
+            {
+                Console.WriteLine("Enter your choice: ");
+                string choice = Console.ReadLine();
+                if (Enum.TryParse(choice, out operation))
+                {
+                    return operation - 1;
+                }
+                Console.WriteLine("Invalid choice");
+            }
+        }
+
+        public static void PrintAccount(List<BankAccount> accounts)
+        {
+            Console.WriteLine("Enter Account Number: ");
+            int accountNumber = int.Parse(Console.ReadLine());
+            BankAccount account = FindAccount(accounts, accountNumber);
+            if (account == null)
+            {
+                Console.WriteLine("Account not found");
+                return;
+            }
+            Console.WriteLine($"Account Number: {account.AccountNumber}");
+            Console.WriteLine($"Name: {account.Name}");
+            Console.WriteLine($"Balance: {account.Balance}");
+            Console.WriteLine($"ID: {account.ID}");
+        }
+       
+   
+        public static BankAccount FindAccount(List<BankAccount> accounts, int accountNumber)
+        {
+            foreach (var account in accounts)
+            {
+                if (account.AccountNumber == accountNumber)
+                {
+                    return account;
+                }
+            }
+            return null;
+        }
+
+        public static void SimulateTimeShift(List<BankAccount> accounts)
+        {
+            foreach (var account in accounts)
+            {
+                account.ShiftYear();
+                account.UpdateInterest();
+                account.UpdateBalance();
+            }
+        }
+
+        public static void Deposit(List<BankAccount> accounts)
```

Comment:
Reviewed, the function doesn't make any another actions, which could be wrapped in one more additional function.
Otherwise, I would like to ask you to you clarify specific problematic code rows